src/main/docker/atlassian.yml

```diff
@@ -1,52 +1,90 @@
-version: "2"
+# ----------------------------------------------------------------------------------------------------------------------
+# Atlassian Setup
+# ----------------------------------------------------------------------------------------------------------------------
+
 services:
     jira:
-        container_name: artemis_jira
+        container_name: artemis-jira
+        hostname: jira
+        extra_hosts:
```

Comment:
[This is the official docker way for Linux systems to be able to use `host.docker.internal` inside these containers.](https://github.com/moby/moby/pull/40007#issuecomment-578729356)

In this case this would be necessary to configure Jira to talk to services on the docker host as localhost would just resolve to the Jira container itself and not the docker host.

Not sure where add the link above in the docs or in code as comment maybe?!

Reply:
Currently, the `atlassian.yml` is just used for development reasons and we don't have a development version of it yet with overrides.
In general, if everything is running inside Docker than you don't need to communicate to the host. But not setting this up is definitely a pitfall which many developers on Linux will face at some time, so I would like to keep it for Developers.

Reply:
Perfect. Then everything is fine :)